### DISCUSSION TITLE
The drone code

### DISCUSSION THREAD

**Discussion by MMWAVEMJK** [jyjblrd/Mocap-Drones#6] [Q&A] [answered]
Hello, How can i access to your drone code?
I want at first make drone by my self and do similar project.I will be happy if you help me.
Thanks

**Comment by jyjblrd** [comment, marked as answer]
Hey, the code that is run on the drone's esp32 microcontroller is here: [receiver_esp32.ino](https://github.com/jyjblrd/Mocap-Drones/blob/main/receiver_esp32/receiver_esp32.ino).

There is also a f3 brushed flight controller which controls the motors. More details can be found here: [https://joshuabird.com/blog/post/mocap-drones](https://joshuabird.com/blog/post/mocap-drones)

**Reply by Ivoferrer9** [reply]
Is the beta flight set to acro or is it in a different mode?

**Reply by jyjblrd** [reply]
It is in Angle (self level) mode. This means that the rc input gives the acceleration of the drone.
